content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,69 @@
+# Machine Learning Model Registry
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Different models can be discovered, audited,
+shared, tested, and deployed from here. DVC works on top of Git to enable these
+capabilities on your existing ecosystem. And other tools from
+[Iterative](https://iterative.ai/) provide even better features and usability.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+Model catalogs give your team key collaborative capabilities:
+
+- Store, organize, and track the [versions] of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to any data artifact, from early-stage [experiment]
+  results to production-ready models.
+- Save performance [metrics and plots] to quickly evaluate and compare models.
+- An interface to access specific model versions from other systems (e.g. for
+  CI/CD)
+
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[experiment]: /doc/user-guide/experiment-management
+[metrics and plots]: /doc/start/metrics-parameters-plots
+
+Many of these benefits are built into DVC: Your [modeling process] and
+performance metadata become **codified** in Git-based <abbr>DVC projects</abbr>.
+This makes it possible to manage them with standard GitOps workflows along with
+code. Large model files are stored separately and can be pushed to [remote
+storage], an efficient and scalable access point.
+
+The relevant outputs of these model development projects can then be
+**packaged** together with their metadata, and published in a [global Git
+repository]. With DVC, existing model files stored remotely are reused, so
+building the registry is fast. A connection to the original projects is
+preserved, so you can always get the full lineage and provenance information.
+
+DVC repos are already great for [sharing] data and models. They can also help
```

Comment:
It's not like repos help get us models - it's DVC provides APIs / tools to get models described in repos ... that are natively compatible with CI systems (since DVC works on top git) - it's "Git native" to some extent :)

Reply:
Maybe this part is somewhat unclear? I'm still thinking mainly on a pure DVC model registry at this point (imported artifacts from other repos). You could `dvc get` models directly from this kind of registry or integrate them with `dvc.api.open`...